_dashboards/branding.md

```diff
@@ -72,29 +72,27 @@ logo:
 applicationTitle: "My custom application"
 ```
 
-We recommend linking to images that are hosted on a web server, but if you really want to use locally hosted images, save your images inside `src/core/server/core_app/assets`, and then configure `opensearch_dashboards.yml`. You can access locally stored images through the `ui` folder.
+We recommend linking to images that are hosted on a web server, but if you really want to use locally hosted images, save your images inside `assets`, and then configure `opensearch_dashboards.yml` to use the correct paths. You can access locally stored images through the `ui/assets` folder.
 
 The following example assumes the default port of 5601 that Dashboards uses and demonstrates how to link to locally stored images.
 
 ```yml
 logo:
-  defaultUrl: "https://localhost:5601/ui/my-own-image.svg"
-  darkModeUrl: "https://localhost:5601/ui/my-own-image.svg"
+  defaultUrl: "https://localhost:5601/ui/assets/my-own-image.svg"
+  darkModeUrl: "https://localhost:5601/ui/assets/my-own-image.svg"
 mark:
-  defaultUrl: "https://localhost:5601/ui/my-own-image2.svg"
-  darkModeUrl: "https://localhost:5601/ui/my-own-image2.svg"
+  defaultUrl: "https://localhost:5601/ui/assets/my-own-image2.svg"
+  darkModeUrl: "https://localhost:5601/ui/assets/my-own-image2.svg"
 # loadingLogo:
 #   defaultUrl: ""
 #   darkModeUrl: ""
 # faviconUrl: ""
 applicationTitle: "My custom application"
 ```
 
-We don't recommend this workaround because new versions of Dashboards would revert all customized elements back to default branding elements, and you would have to re-upload your assets to access them again.
-
 ## Sample configuration
 
-The following configuration enables the security plugin within OpenSearch Dashboards and uses custom branding elements to replace the OpenSearch logo and application title.
+The following configuration enables SSL within OpenSearch Dashboards and uses custom branding elements to replace the OpenSearch logo and application title.
```

Comment:
NIT: The security plugin is a good specification here because it enables the security plugin. However, they can still enable SSL within OpenSearch Dashboards.

Reply:
Added back in.